pymeasure/display/widgets.py

```diff
@@ -1129,6 +1129,10 @@ def _layout(self):
         update_hbox.addWidget(self.update_button)
         f_layout.addRow("Auto-update (off / slow / fast)", update_hbox)
 
+    def change_size_for_floating(self):
+        print(self.parent().isFloating())
```

Comment:
please use log.warning()|error|etc instead of print. :-)

Reply:
Yes, I will indeed, it is now there just for (temporary) debugging purpose.

Reply:
You could do it with `log.debug()`, use `logging.basicConfig(level=logging.DEBUG)` in your testing script/console. Then debug to your heart's content, maybe adapt the debug output for maximum usefulness, and then...just leave the log in the code, when you need to debug again in the future. :-)